src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/MinMaxTable.cs

```diff
@@ -23,5 +28,44 @@ public MinMaxTableFunction(bool isMin)
         {
             _delegationCapability = isMin ? DelegationCapability.Min : DelegationCapability.Max;
         }
+
+        public override bool CheckInvocation(TexlNode[] args, DType[] argTypes, IErrorContainer errors, out DType returnType, out Dictionary<TexlNode, DType> nodeToCoercedTypeMap)
+        {
+            Contracts.AssertValue(args);
+            Contracts.AssertValue(argTypes);
+            Contracts.Assert(args.Length == argTypes.Length);
+            Contracts.AssertValue(errors);
+            Contracts.Assert(MinArity <= args.Length && args.Length <= MaxArity);
+
+            // The return type will be the result type of the expression in the 2nd argument
+            var fArgsValid = true;
+            returnType = argTypes[1];
+            nodeToCoercedTypeMap = null;
+
+            // Coerce everything except date/times to numeric.
+            for (var i = 1; i < argTypes.Length; i++)
```

Comment:
there should be only 2 args to this function, I don't think the loop is needed

Reply:
removed